MC/run/ANCHOR/anchorMC.sh

```diff
@@ -0,0 +1,186 @@
+ALIEN_JDL_LPMPRODUCTIONTAG=$ALIEN_JDL_LPMANCHORPRODUCTION
+
+# ZDC causes issues for sim
+#export ALIEN_JDL_WORKFLOWDETECTORS=ITS,TPC,TOF,FV0,FT0,FDD,MID,MFT,MCH,TRD,EMC,PHS,CPV,HMP,ZDC,CTP
+export ALIEN_JDL_WORKFLOWDETECTORS=ITS,TPC,TOF,FV0,FT0,FDD,MID,MFT,MCH,TRD,EMC,PHS,CPV,HMP,CTP
+
+### async_pass.sh
+DPGRECO=$O2DPG_ROOT/DATA/production/configurations/asyncReco/async_pass.sh
+DPGSETENV=$O2DPG_ROOT/DATA/production/configurations/asyncReco/setenv_extra.sh
+
+if [[ -f async_pass.sh ]]; then
+    chmod +x async_pass.sh
+    DPGRECO=./async_pass.sh
+else
+    cp -v $DPGRECO .
+fi
+
+if [[ ! -f setenv_extra.sh ]]; then
+    DPGSETENV=${DPGRECO%/*}/setenv_extra.sh
+    [[ -f ${DPGSETENV} ]] && cp ${DPGSETENV} . || cp ${DPGSETENV} .
+fi
+
+echo "[INFO alien_async_pass.sh] Setting up DPGRECO to ${DPGRECO}"
+
+#settings that are MC-specific
+sed -i 's/GPU_global.dEdxUseFullGainMap=1;GPU_global.dEdxDisableResidualGainMap=1/GPU_global.dEdxSplineTopologyCorrFile=splines_for_dedx_V1_MC_iter0_PP.root;GPU_global.dEdxDisableTopologyPol=1;GPU_global.dEdxDisableGainMap=1;GPU_global.dEdxDisableResidualGainMap=1;GPU_global.dEdxDisableResidualGain=1/' setenv_extra.sh
+### ???
+
+chmod +x async_pass.sh
+
+# take out line running the workflow (if we don't have data input)
+[ ${CTF_TEST_FILE} ] || sed -i '/WORKFLOWMODE=run/d' async_pass.sh
+
+# create workflow ---> creates the file that can be parsed
+export IGNORE_EXISTING_SHMFILES=1
+touch list.list
+
+./async_pass.sh ${CTF_TEST_FILE:-""} 2&> async_pass_log.log
+RECO_RC=$?
+
+echo "RECO finished with ${RECO_RC}"
+if [ "${NO_MC}" ]; then
+  return ${RECO_RC} 2>/dev/null || exit ${RECO_RC} # optionally quit here and don't do MC (useful for testing)
+fi
+
+ALIEN_JDL_LPMPRODUCTIONTAG=$ALIEN_JDL_LPMPRODUCTIONTAG_KEEP
+echo "Setting back ALIEN_JDL_LPMPRODUCTIONTAG to $ALIEN_JDL_LPMPRODUCTIONTAG"
+
+# now create the local MC config file --> config-config.json
+${O2DPG_ROOT}/UTILS/parse-async-WorkflowConfig.py
+
+# check if config reasonably created
+if [[ `grep "o2-ctf-reader-workflow-options" config-json.json 2> /dev/null | wc -l` == "0" ]]; then
+  echo "Problem in anchor config creation. Stopping."
+  exit 1
+fi
+
+# -- CREATE THE MC JOB DESCRIPTION ANCHORED TO RUN --
+
+NWORKERS=${NWORKERS:-8}
+MODULES="--skipModules ZDC"
+SIMENGINE=${SIMENGINE:-TGeant4}
+SIMENGINE=${ALIEN_JDL_SIMENGINE:-${SIMENGINE}}
+NTIMEFRAMES=${NTIMEFRAMES:-50}
+NSIGEVENTS=${NSIGEVENTS:-22}
+
+SPLITID=${SPLITID:-0}
+PRODSPLIT=${PRODSPLIT:-100}
+CYCLE=${CYCLE:-0}
+# let SEED=$SPLITID+$CYCLE*$PRODSPLIT
+SEED=${ALIEN_PROC_ID}
+
+# create workflow
+# THIS NEEDS TO COME FROM OUTSIDE
+# echo "$" | awk -F' -- ' '{print $1, $3}'
+
+baseargs="-tf ${NTIMEFRAMES} --split-id ${SPLITID} --prod-split ${PRODSPLIT} --cycle ${CYCLE} --run-number ${RUNNUMBER}"
+baseargs_alt="-col pp -eCM 13600 -tf ${NTIMEFRAMES} --split-id ${SPLITID} --prod-split ${PRODSPLIT} --cycle ${CYCLE} --run-number ${RUNNUMBER}"
```

Comment:
This should not be there. I had it in order to test various different versions with the same `anchorMC.sh`